### PR TITLE
num cuts as attribute

### PR DESCRIPTION
**Pull request by matbesancon**

### CODE REVIEW

**Comment by matbesancon** [comment]
not sure what happened with precompiling here

File: src/MOI_wrapper.jl [hunk 1]
```diff
@@ -136,3 +136,14 @@ end
 function MOI.get(opt::Optimizer, ::MOI.NodeCount)
     return MOI.get(opt.oa_model, MOI.NodeCount())
 end
+
+"""
+    NumberOfCuts
+
+Number of cuts added to the outer approximation model.
+"""
+struct NumberOfCuts <: MOI.AbstractOptimizerAttribute end
```
Comment (blegat) [inline src/MOI_wrapper.jl:145]:
```suggestion
struct NumberOfCuts <: MOI.AbstractModelAttribute end
```

File: src/MOI_wrapper.jl [hunk 2]
```diff
@@ -136,3 +136,14 @@ end
...
+
+function MOI.get(opt::Optimizer, ::NumberOfCuts)
```
Comment (blegat) [inline src/MOI_wrapper.jl:147]: Implement `is_set_by_optimize` and `attribute_value_type`, you can take example on `MOI.NodeCount` for instance

File: src/MOI_wrapper.jl (hunk 1 above)
Comment (matbesancon) [reply src/MOI_wrapper.jl:145]: is that a model attribute if it's set by optimizing? It's not tied to the model itself but the solving process

File: src/MOI_wrapper.jl (hunk 2 above)
Comment (matbesancon) [reply src/MOI_wrapper.jl:147]: these two methods are not implemented for NodeCount here no?

Comment (matbesancon) [reply src/MOI_wrapper.jl:147]: ah ok I saw what you meant, done

**Comment by chriscoey** [comment]
Thanks for adding this @matbesancon, looks good!

**Comment by chriscoey** [comment]
I could have sworn I selected squash when I merged that, but doesn't look like it.

Anyway, I invited you as a collaborator @matbesancon. If you want to tag a new minor version, go ahead!

**Comment by matbesancon** [comment]
perfect thanks!

**Pull request merged**